ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/3_Manage_Complexity/3_1_Military_Time_Validator/src/index.spec.ts

```diff
@@ -1,5 +1,45 @@
+import validateMilitaryTime from './index';
 
 describe('military time validator', () => {
+    it.each([
+        ['01:12 | 14:32', false],
+        ['01:12/14:32', false],
+        ['01:12, 14:32', false],
+        ['01:12 14:32', false],
+        ['01:12-14:32', true],
+    ])('knows what that the range should be separated by a `-`', (timeRange, expected) => {
+        const output = validateMilitaryTime(timeRange);
 
+        expect(output).toBe(expected);
+    });
 
+    it.each([
+        ['01:12 - 14:32', true],
+        ['-01:12 14:32', false],
+        ['01:12 14:32-', false],
+    ])('knows what that the range should have two times', (timeRange, expected) => {
+        const output = validateMilitaryTime(timeRange);
+
+        expect(output).toBe(expected);
+    });
+
+    it.each([
+        ['09:82 - 14:32', false],
+        ['01:12 - 52:32', false],
+        ['13:12 - 14:72', false],
+    ])('knows what that the max hour is 23 and the max minute is 59 in range %s', (timeRange, expected) => {
+        const output = validateMilitaryTime(timeRange);
+
+        expect(output).toBe(expected);
+    });
+
+    it.each([
+        ['09:82 - 07:32', false],
+        ['23:12 - 00:01', false],
+        ['13:12 - 04:12', false],
```

Comment:
it would be great to add test for case that is covered in code - same hour in first and second part
like `13:23 - 13:12`